<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ErrorShower</class>
 <widget class="QWidget" name="ErrorShower">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>468</width>
    <height>312</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>.PingFang SC0</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>崩溃报告</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0" colspan="4">
    <widget class="QTextBrowser" name="textBrowser"/>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>关闭</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>反馈给作者...</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>崩溃原因：</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <family>.PingFang SC0</family>
       <pointsize>48</pointsize>
      </font>
     </property>
     <property name="text">
      <string>❌</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>.PingFang SC0</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>uClock遇到了严重的问题，导致程序崩溃</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
